### PR TITLE
OSCI-4011: display linked Errata advisories

### PR DESCRIPTION
**Pull request by Andrei-Stepanov**
Signed-off-by: Andrei Stepanov <astepano@redhat.com>

### CODE REVIEW

**Comment by Andrei-Stepanov** [comment]
PR can be merged. @mgrabovsky please review.

File: src/components/ArtifactDetailedInfo.tsx [hunk 1]
```diff
@@ -91,177 +279,103 @@ const ArtifactDetailedInfoKojiBuild: React.FC<
...
-                    <DescriptionListGroup>
-                        <DescriptionListTerm>Commit time</DescriptionListTerm>
-                        <DescriptionListDescription
-                            className={styles['buildInfoTimestamp']}
-                        >
-                            {commitTimeWithTz}
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                </DescriptionList>
+                <NoData show={_.every([!haveData, !loadingCurrentState])} />
```
Comment (mgrabovsky) [inline src/components/ArtifactDetailedInfo.tsx:299]: This seems unnecessarily contrived. Why not just this?
```suggestion
                <NoData show={!haveData && !loadingCurrentState} />
```

File: src/components/ArtifactDetailedInfo.tsx [hunk 2]
```diff
@@ -91,177 +279,103 @@ const ArtifactDetailedInfoKojiBuild: React.FC<
...
+                        rel="noopener noreferrer"
+                    >
+                        {adv.advisory_name}
+                    </a>
+                </Td>
+            </Tr>,
+        );
+    }
+    return (
+        <>
```
Comment (mgrabovsky) [inline src/components/ArtifactDetailedInfo.tsx:360]: You can drop the wrapper fragment here.

File: src/components/ArtifactDetailedInfo.tsx [hunk 3]
```diff
@@ -91,177 +279,103 @@ const ArtifactDetailedInfoKojiBuild: React.FC<
...
-                        <DescriptionListTerm>Commit time</DescriptionListTerm>
-                        <DescriptionListDescription
-                            className={styles['buildInfoTimestamp']}
-                        >
-                            {commitTimeWithTz}
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                </DescriptionList>
+                <NoData show={_.every([!haveData, !loadingCurrentState])} />
+                <LoadingData show={loadingCurrentState} />
```
Comment (mgrabovsky) [inline src/components/ArtifactDetailedInfo.tsx:300]: Same here, I would prefer
```suggestion
                {loadingCurrentState && <LoadingData />}
```

**Review by mgrabovsky — COMMENTED** [review]
Looks good overall. I have only a couple of suggestions for improvements.

File: src/components/ArtifactDetailedInfo.tsx (hunk 2 above)
Comment (Andrei-Stepanov) [reply src/components/ArtifactDetailedInfo.tsx]: Fixed as suggested

File: src/components/ArtifactDetailedInfo.tsx [hunk 4]
```diff
@@ -290,14 +404,6 @@ const ArtifactDetailedInfoModuleBuild: React.FC<
...
 
     if (loading) {
-        return (
-            <Flex className="pf-u-p-lg">
-                <FlexItem>
-                    <Spinner className="pf-u-mr-md" size="md" /> Loading build
-                    info…
-                </FlexItem>
-            </Flex>
-        );
```
[the remark this answers is not in the document]
Comment (Andrei-Stepanov) [reply src/components/ArtifactDetailedInfo.tsx:300]: It was moved to component: `LoadingData`

File: src/components/ArtifactDetailedInfo.tsx (hunk 3 above)
Comment (Andrei-Stepanov) [reply src/components/ArtifactDetailedInfo.tsx:300]: I merge this PR. All other concers are fixed.
For this topic I opened ticket: https://issues.redhat.com/browse/OSCI-4404

Comment (Andrei-Stepanov) [reply src/components/ArtifactDetailedInfo.tsx:300]: Ah, code review is still in pending. Please approve.

File: src/components/ArtifactDetailedInfo.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (mgrabovsky) [reply src/components/ArtifactDetailedInfo.tsx:300]: But that component is not used in `ArtifactDetailedInfoModuleBuild`. Hence it show the "No build information available" message until it loads.

[the remark this answers is not in the document]
Comment (Andrei-Stepanov) [reply src/components/ArtifactDetailedInfo.tsx]: Ah, right. Updated.

**Review by mgrabovsky — APPROVED** [review]

**Pull request merged**